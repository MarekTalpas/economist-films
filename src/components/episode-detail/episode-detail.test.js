// @flow
import React from 'react';
import renderer from 'react-test-renderer';
import EpisodeDetail from './episode-detail';

test('EpisodeDetail renders correctly', () => {
  const tree : string = renderer.create(<EpisodeDetail />).toJSON();
  expect(tree).toMatchSnapshot();
});
