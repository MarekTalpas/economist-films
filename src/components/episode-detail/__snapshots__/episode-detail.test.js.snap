exports[`test EpisodeDetail renders correctly 1`] = `
<div
  className="episode-detail">
  <div
    className="episode-detail__details">
    <div
      className="episode-watch">
      <div
        className="episode-watch__video"
        style={
          Object {
            "background": "#000 url(\"\") center/cover no-repeat",
          }
        } />
      <button
        className="episode-watch__watchnow-button">
        Watch now
      </button>
    </div>
    <div
      className="episode-description">
      <h3
        className="episode-description__date">
        ...loading...
      </h3>
      <h1
        className="episode-description__title">
        ...loading...
      </h1>
      <p
        className="episode-description__text">
        ...loading...
      </p>
    </div>
  </div>
  <div
    className="episode-detail__cards">
    <div
      className="episode-cards">
      <h1
        className="episode-cards__title">
        Recommended
      </h1>
      <div
        className="episode-cards__wrapper" />
    </div>
  </div>
</div>
`;
